contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -816,6 +822,20 @@ public void runPushStore(Props props, String url, String dataDir) throws Excepti
 
         log.info("Push starting for cluster: " + url);
 
+        // CDN distcp
+        boolean cdnEnabled = props.getBoolean(VoldemortBuildAndPushJob.PUSH_CDN_ENABLED, false);
+        String modifiedDataDir = new Path(dataDir).makeQualified(FileSystem.get(new JobConf())).toString();
+
+        if (cdnEnabled) {
+            if (modifiedDataDir.matches(".*hdfs://.*:[0-9]{1,5}/.*")) {
```

Comment:
`hdfs`, `webhdfs`, `swebhdfs` will match this regex. Are we using any other protocols?